cmd/buildkitd/debug.go

```diff
@@ -40,6 +43,7 @@ func setupDebugHandlers(addr string) error {
 	m.Handle("/debug/cache/all", http.HandlerFunc(handleCacheAll))
 	m.Handle("/debug/cache/lookup", http.HandlerFunc(handleCacheLookup))
 	m.Handle("/debug/cache/store", http.HandlerFunc(handleDebugCacheStore))
+	m.Handle("POST /debug/cache/load", http.HandlerFunc(handleCacheLoad))
```

Comment:
Wonder if we could have a `/debug/cache/import` endpoint that could use the same CSV attributes as `cache-from` to make this easier? Like `type=registry,ref=foo/bar:cache`